sidebars.js

```diff
@@ -66,19 +66,22 @@ module.exports = {
   'Contributing to OpenRefine': [
 
     {
-      type: 'category',
+      type: 'link',
+      label: 'Getting-started',
+      href: 'technical-reference/contributing',
+    },
```

Comment:
It looks like the syntax you are using to create simple entries (not categories) is not working, because [the logs](https://app.netlify.com/sites/openrefine-website/deploys/65294d3af9fff500087eb8ff) report an error on every page of the website (likely meaning that the `href` links you introduced do not work).

Instead of using this more verbose syntax (`{ type: 'link', label: … }`), why not simply use the more concise way of having only the path to the file (such as `'technical-reference/contributing'`)? You can see an example of that here: https://github.com/OpenRefine/openrefine.org/pull/252/files